internal/model/team.go

```diff
@@ -64,3 +64,7 @@ func (t Team) IsHW() bool {
 func (t Team) IsLayNewOrdersTeam() bool {
 	return t.Name == "Lay Team - New Deputy Orders"
 }
+
+func (t Team) IsFullLayTeam() bool {
+	return t.Name == "Lay Deputy Team"
```

Comment:
I've used this way of identifying it in the `IsLay()` method - the "normal" teams will have their ID as the Selector
```suggestion
	return t.Selector == "lay-team"
```